src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -156,6 +156,7 @@ protected void updateItem(final TableRow item, final boolean empty) {
                      label.setUnderline(item.isUnderlined());
                      label.setTooltip(new Tooltip(notes));
                      this.setGraphic(label);
+                     workTableTreeView.refresh();
```

Comment:
can you try to move the Circle into this update method? so that for ProjectRows the circle is set as graphic and the project name as text.
- As the text cannot be underlined (for work projects) not sure if we want to indicate work items another way
   - maybe we underline the time of the project instead?

Reply:
Moved the text and the circle to the Update Method. 
For the work maybe add an extra column ?

Reply:
lets just put a border on the circle for now - when it is work
circle.setStroke(Color.BLACK);

Reply:
I underlined the Time column to indicate that it is a work item. I can also add thie cirlce border

Reply:
can leave it like this then. just thought we could save the hastle with custom cell renderer - but as it is already implemented now we can keep it